### ISSUE TITLE
Unable to fire syndicate sniper rifle

### ISSUE THREAD

**Issue by PerfectTangent** [unitystation/unitystation#7052]
### Detailed description of bug

Spawned a syndicate sniper bundle as a nuclear operative and tried firing it. Got the message 'User Authentication Failed' despite having a nuclear operative ID. This error was appearing every time I tried to fire the gun:

```
System.ArgumentNullException: Value cannot be null.
Parameter name: first
  at System.Linq.Enumerable.Intersect[TSource] (System.Collections.Generic.IEnumerable`1[T] first, System.Collections.Generic.IEnumerable`1[T] second) [0x0000d] in <351e49e2a5bf4fd6beabb458ce2255f3>:0 
  at Systems.Clearance.ClearanceCheckable.HasClearance (System.Collections.Generic.IEnumerable`1[T] requesterClearance) [0x00027] in <ff92e220c299495dbda6c97f44426ebd>:0 
  at Systems.Clearance.ClearanceCheckable.HasClearance (UnityEngine.GameObject player) [0x0005d] in <ff92e220c299495dbda6c97f44426ebd>:0 
  at Weapons.IDLockedPin.ServerBehaviour (AimApply interaction, System.Boolean isSuicide) [0x0000c] in <ff92e220c299495dbda6c97f44426ebd>:0 
  at Weapons.Gun.ServerPerformInteraction (AimApply interaction) [0x0003c] in <ff92e220c299495dbda6c97f44426ebd>:0 
  at Messages.Client.Interaction.RequestInteractMessage.ProcessInteraction[T] (T interaction, UnityEngine.GameObject processorObj, System.Type ComponentType) [0x000c3] in <ff92e220c299495dbda6c97f44426ebd>:0 
  at Messages.Client.Interaction.RequestInteractMessage.Process (Messages.Client.Interaction.RequestInteractMessage+NetMessage msg) [0x0029f] in <ff92e220c299495dbda6c97f44426ebd>:0 
  at Messages.GameMessageBase`1[T].Process (Mirror.NetworkConnection sentBy, T msg) [0x00000] in <ff92e220c299495dbda6c97f44426ebd>:0 

```

**Comment by Aranclanos** [comment]
thanks for the error log, for the moment dont use your PDA in your ID slot for vendors, emitters and weapons until it gets fixed

**Comment by Aranclanos** [comment]
Fixed in #7086
